### ISSUE TITLE
WSL tutorials should clearly state they are Windows 11 only

### ISSUE THREAD

**Issue by dotnetCarpenter** [MicrosoftDocs/WSL#1259]
I'm using WSL2  Ubuntu 21.04 on Windows 10 with a x86_64 5.10.16.3-microsoft-standard-WSL2 kernel.

I need to run [a python3 script](https://github.com/yasoob/Baseline-JPEG-Decoder) that uses `python3-tk` .

While I can install `python3-tk` via `apt`, I get the following error: `_tkinter.TclError: no display name and no $DISPLAY environment variable`.

So I found [someone needing to setup `$DISPLAY` on WSL2](https://askubuntu.com/questions/1299323/how-to-set-up-display-variable-for-wsl2-of-ubuntu-20), then I found a [Microsoft Tech article about installing VcXsrv Windows X Server](https://techcommunity.microsoft.com/t5/windows-dev-appconsult/running-wsl-gui-apps-on-windows-10/ba-p/1493242) and in that article it said that support for running GUI Apps will soon be released. Hence, I found [this article](https://docs.microsoft.com/en-us/windows/wsl/tutorials/gui-apps).

I'm on the Insider Program but got bumped from the Beta Channel to Release Preview Channel, because I only have a TPM 1.2 chip and apparently everything earlier than Release Preview Channel is Windows 11 only.

[Run Linux GUI apps on the Windows Subsystem for Linux (preview)](https://docs.microsoft.com/en-us/windows/wsl/tutorials/gui-apps) fails to mention that this feature is only available in Windows 11! The article is not old, its from April this year (2021).

**Please update the article to state that this is Windows 11 only or provide a way for Windows 10 users to get  Windows 10 Insider Preview build 21362!**

Since I will never be able to run Windows 11 on my laptop it would waste 100% less of my time if all articles on https://docs.microsoft.com/en-us/windows/wsl/tutorials/ would clearly state that they are Windows 11 only or provide work-arounds.

**Thanks!**

PS. `winver` says I have Version 21H2 (OS Build 19044.1165) - there should to be a way to copy this information (I had to write down what the dialog said!).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9766c3d7-c8ac-d361-3d34-eadaacee283c
* Version Independent ID: 03d806dd-56d9-f230-b7f8-b6f583168247
* Content: [Run Linux GUI apps with WSL](https://docs.microsoft.com/en-us/windows/wsl/tutorials/gui-apps)
* Content Source: [WSL/tutorials/gui-apps.md](https://github.com/MicrosoftDocs/wsl/blob/master/WSL/tutorials/gui-apps.md)
* Product: **dev-environment**
* Technology: **windows-subsystem-for-linux**
* GitHub Login: @craigloewen-msft
* Microsoft Alias: **crloewen**

**Comment by dzid26** [comment]
The article is confusing. 
Seems like Windows 10 Insider Preview 10.0.**21**364.1000  was available a couple of months ago in the Dev Insider channel, but it isn't anymore. 
It's also not available as a standalone [ISO](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewiso).

So I don't see a way to get the WSL gui on Win10 anymore.

**Comment by dotnetCarpenter** [comment]
@dzid26 I've successfully used https://techcommunity.microsoft.com/t5/windows-dev-appconsult/running-wsl-gui-apps-on-windows-10/ba-p/1493242 on my system (disregard the xfce4 stuff, unless you are planning to use that desktop environment).

>To avoid having to run that command every time that WSL is launched, you can include the command at the end of the /etc/bash.bashrc file:

I also skipped that ☝️  part and instead added `export DISPLAY` to my `.profile`. I'm using Ubuntu BTW

```
export DISPLAY="`sed -n `'s/nameserver //p' /etc/resolv.conf`:0"
```

But you are right about the article. It should be fixed. False hope and all that :(
